rust/stub/src/main.rs

```diff
@@ -73,15 +73,15 @@ struct EmbeddedConfiguration {
 }
 
 /// Extract a string, stored as UTF-8, from a PE section.
-fn extract_string(file_data: &[u8], section: &str) -> Result<CString16> {
```

Comment:
I've renamed `file_data` to `pe_data`, because these functions now assume that the PE image is already loaded. This makes a difference when we extract section data.